match_against_all_templates/equivalentwords.txt

```diff
@@ -0,0 +1,43 @@
+acknowledgement,acknowledgment
```

Comment:
I see this file in every folder. Please move this to resources/ folder and use it across the modules

Reply:
Similarly test.txt